src/goto-programs/goto_program_template.h

```diff
@@ -216,8 +216,13 @@ class goto_program_templatet
     }
 
     //! Uniquely identify an invalid target or location
+    #if (defined _MSC_VER && _MSC_VER <= 1800)
```

Comment:
Can you add an explanation for 1800 and the variable name? Something like "_MSC_VER is a version of the Microsoft 'cl' compiler, 1800 corresponds to the Visual Studio 2013 version"